doc/manual_rest.rst

```diff
@@ -50,22 +51,24 @@ Usage help is available:
           --cleanup-cache              auto remove old cache directories
       -h, --help                       help for restic
           --json                       set output mode to JSON for commands that support it
-          --key-hint key               key ID of key to try decrypting first (default: $RESTIC_KEY_HINT)
+          --key-hint string            key ID of key to try decrypting first (default: $RESTIC_KEY_HINT)
```

Comment:
Seems to me you are undoing a lot of recent changes in `manual_rest.rst`. Maybe you did something wrong when rebasing to the current master?

Reply:
Yeah, looks like it.  

The rebase gives me merge conflicts of the entire chunk of command flags, due to the spacing.  (this happens with structs in Go as well.  :grimacing:)

It might be easier to just re-generate the command output.